feat(starr): Update the DV information for the Infuse player, which appears to support the CMv4.0 metadata (#2343)
TRaSH-Guides/Guides: includes/cf/dv-info-profiles.md

    - <sup>(*1*) *PLEX for AppleTV and Plex with Infuse will only play profile 5 correctly if CMv2.9 is used*</sup>
    - <sup>(*2*) *Neither Infuse nor PLEX for AppleTV will deliver real Dolby Vision with Profile 7.*</sup>

>     - <sup>(*3*) *On PLEX for AppleTV, it won't deliver real Dolby Vision with Profile 8 and will only play the HDR10 fallback if CM2.9 is used. Otherwise, you may encounter a black screen*<br>***“although your TV will incorrectly indicate that it is playing DV”***.<br>*Infuse 7.7.2+ includes expanded support for Dolby Vision (Profile 8). This includes support for files which contain CMv4.0 metadata, and samples which were previously playing with a black screen, falling back HDR, or exhibiting other various playback issues. [SOURCE](https://community.firecore.com/t/infuse-7-7-2-now-available/48208){:target="\_blank" rel="noopener noreferrer"}*<br>***“Ensure you set the Extended Dolby Vision settings to Limited (prefer accuracy), Convert P8 to P5 (when possible), and play other P8 as HDR (output will switch to either DoVi or HDR depending on the video)”***</sup>

Reviewer: ```suggestion
    - <sup>(*3*) *On PLEX for AppleTV, it won't deliver real Dolby Vision with Profile 8 and will only play the HDR10 fallback if CM2.9 is used. Otherwise, you may encounter a black screen*<br>***“although your TV will incorrectly indicate that it is playing DV”***.<br>*Infuse 7.7.2+ includes expanded support for Dolby Vision (Profile 8). This includes support for files which contain CMv4.0 metadata, and samples which were previously playing with a black screen, falling back to HDR, or exhibiting other various playback issues. [SOURCE](https://community.firecore.com/t/infuse-7-7-2-now-available/48208){:target="\_blank" rel="noopener noreferrer"}*<br>***“Ensure you set the Extended Dolby Vision settings to Limited (prefer accuracy), Convert P8 to P5 (when possible), and play other P8 as HDR (output will switch to either DoVi or HDR depending on the video)”***</sup>
```

??? info "Dolby Vision Versions - CMv2.9 and CMv4.0 - [Click to show/hide]"

>     !!! bug "It is too technical to explain in detail, so I'll keep it short with verified facts: <br>PLEX for AppleTV only supports CMv2.9<br>Plex with Infuse 7.7.2+ includes expanded support for Dolby Vision (Profile 8). This includes support for files which contain CMv4.0 metadata, and samples which were previously playing with a black screen, falling back HDR, or exhibiting other various playback issues. [SOURCE](https://community.firecore.com/t/infuse-7-7-2-now-available/48208)"

Reviewer: this seems like duplicate information 🤔

Author: ![image](https://github.com/user-attachments/assets/b4345cc1-30b6-4e0c-99b6-e68404e7e23a)
because they are used for two different questions
Question 1. Dolby Vision Versions - CMv2.9 and CMv4.0
Question 2. Dolby Vision Profiles

So it depends on which question they look for.

    - <sup>(*1*) *PLEX for AppleTV and Plex with Infuse will only play profile 5 correctly if CMv2.9 is used*</sup>
    - <sup>(*2*) *Neither Infuse nor PLEX for AppleTV will deliver real Dolby Vision with Profile 7.*</sup>

>     - <sup>(*3*) *PLEX for AppleTV will only play profile 8 correctly if CMv2.9 is used. However, we have also received reports that in some cases, it will fall back to HDR10, or you may encounter a black screen.*<br><br>The following information has been provided by a PLEX for AppleTV/Infuse user:<br>To prevent so called fake DV (“Your TV will incorrectly indicate that it is playing DV”)**<br>“With Infuse ensure you set the Extended Dolby Vision settings to Limited (prefer accuracy), convert P8 to P5 (when possible), and play other P8 as HDR (output will switch to either DoVi or HDR depending on the video)”**</sup>

Reviewer: ```suggestion
    - <sup>(*3*) *PLEX for Apple TV will only play profile 8 correctly if CMv2.9 is used. However, we have also received reports that in some cases, it will fall back to HDR10, or you may encounter a black screen.*<br><br>To prevent your TV from incorrectly indicating that it is playing DV follow the steps provided by an Infuse user:**<br>“With Infuse ensure you set the Extended Dolby Vision settings to Limited (prefer accuracy), convert P8 to P5 (when possible), and play other P8 as HDR (output will switch to either DoVi or HDR depending on the video)”**</sup>
```
